tests/integration/security/mtls_first_party_jwt/strict_test.go

```diff
@@ -64,6 +68,10 @@ func TestMtlsStrictK8sCA(t *testing.T) {
 					ConfigFile: "global-plaintext.yaml",
 					Namespace:  systemNM,
 					Include: func(src echo.Instance, opts echo.CallOptions) bool {
+						// Exclude headless naked service
```

Comment:
I don't really get why we need to exclude these... this is for plaintext?

Reply:
Yes, it can pass the test. I just wanted to test it in reachability test. Will open it